coral/functions/update_archive_display_name_function.py

```diff
@@ -0,0 +1,49 @@
+from arches.app.functions.base import BaseFunction
+from arches.app.models.tile import Tile
+
+SYSTEM_REF_NODEGROUP = "3bdc39f8-9a93-11ea-b807-f875a44e0e11"
+ARCHIVE_SOURCE_NAME_NODEGROUP = "145f9615-9ad2-11ea-b4d3-f875a44e0e11"
+DISPLAY_NAME_NODEGROUP = "b5d2204c-442b-11ef-a57a-0242ac120002"
+
+ARCHIVE_SOURCE_NAME_NODE = "145f961b-9ad2-11ea-bf90-f875a44e0e11"
+RESOURCE_ID_NODE = "3bdc39fc-9a93-11ea-8cee-f875a44e0e11"
+DISPLAY_NAME_NODE = "eb0b2aec-442b-11ef-a57a-0242ac120002"
+
+details = {
+    # "functionid": "",
```

Comment:
function id needs included here or it will not match with the graph

Reply:
added in